### ISSUE TITLE
xy_loss problem

### ISSUE THREAD

**Issue by qianlinjun** [Stinky-Tofu/YOLO_V3#44]
感谢代码分享! 看代码时对loss函数中的xy_loss有点疑问?为什么使用tf.nn.sigmoid_cross_entropy_with_logits这个函数而不是平方差函数计算xy loss, 看了tensorflow sigmoid_cross_entropy_with_logits函数说明如下:
For brevity, let `x = logits`, `z = labels`.  The logistic loss is
        z * -log(sigmoid(x)) + (1 - z) * -log(1 - sigmoid(x))
不太理解后面 (1 - z) * -log(1 - sigmoid(x))这部分的作用

**Comment by Stinky-Tofu** [comment]
@qianlinjun 这个啊，这个就是逻辑回归的公式呀

**Comment by qianlinjun** [comment]
对 用逻辑回归来预测类别标签是0或1概率 我理解没问题, 但是这里真实标签(label_txty)并不是0或1,而是0-1之间, 所以有点疑惑,并不是太理解使用这个函数来计算xy_loss的物理意义

**Comment by Stinky-Tofu** [comment]
@qianlinjun 好像还真是诶，确实有点问题，当时写的时候也是参考了一些其他代码，没太注意到这个。

**Comment by qianlinjun** [comment]
好的, 看到也有人在keras yolo3项目里问了同样的问题,[https://github.com/qqwweee/keras-yolo3/issues/230](url),作者还没有回复

**Comment by Stinky-Tofu** [comment]
@qianlinjun 哈哈，一个人维护的话，这么多问题，作者根本不想回复。可以微信交流18890099926
